web-components/src/mixins/FocusTrapMixin.ts

```diff
@@ -279,6 +280,21 @@ export const FocusTrapMixin = <T extends AnyConstructor<FocusClass & FocusTrapCl
       }
     }
 
+    protected setFocusOnTrigger(triggerElement: HTMLElement) {
+      let deepNestedTriggerElement;
+      if (triggerElement.shadowRoot!) {
```

Comment:
What will happen for a trigger element which is a span (non focusable element)? You can check the example from TPW customization button and check

Reply:
as the correct implementation of menu-overlay there needs to be a focusable element to work it properly, however we have handled the case when focusable element is inside a span then we are extracting it out.